src/components/User/Admins/components/AdminManager.vue

```diff
@@ -113,11 +117,21 @@
 
           <template v-else>
             <button
+              v-if="addNew"
+              class="app__btn"
+              @click="isDeleteMode = false"
+              :disabled="isMaster || formMixin.isDisabled"
+            >
+              {{ "admin-manager.btn-add" | globalize }}
+            </button>
+
+            <button
+              v-else
               class="app__btn"
               @click="isDeleteMode = false"
               :disabled="isMaster || formMixin.isDisabled"
             >
-              {{ addNew ? 'Add' : 'Update' }}
+              {{ "admin-manager.btn-upd" | globalize }}
```

Comment:
why did you add a new button if they are equal?
Use one button
Add v-if for text

Reply:
done